airbyte/_processors/sql/snowflakecortex.py

```diff
@@ -107,9 +107,10 @@ def _get_column_list_from_table(
         This is overridden due to lack of SQLAlchemy compatibility for the
         `VECTOR` data type.
         """
+        normalized_table_name = self.get_sql_table_name(table_name)
         conn: Connection = self.cache.get_vendor_client()
         cursor = conn.cursor()
-        cursor.execute(f"DESCRIBE TABLE {table_name};")
+        cursor.execute(f"DESCRIBE TABLE {normalized_table_name};")
```

Comment:
Normalizing table name beforehand, so query is called with normalized name. For example, if the table name passed was "test-table" it will be normalized to "test_table".